GraphAlgorithms/Program.cs

```diff
@@ -17,271 +12,23 @@ private static void Main(string[] args)
-            }
-            else if (comparison == "vertices-then-edges")
-            {
-                clique = exactFinder.FindWithEdges(graphs[index]);
-            }
-            else throw new NotImplementedException();
-        }
-        else if (algType == "heuristic")
-        {
-            MaxCliqueHeuFinder heuristicFinder = new MaxCliqueHeuFinder();
-            if (comparison == "vertices")
-            {
-                clique = heuristicFinder.Find(graphs[index]);
-            }
-            else if (comparison == "vertices-then-edges")
-            {
-                clique = heuristicFinder.FindWithEdges(graphs[index]);
-            }
-            else throw new NotImplementedException();
-        }
-        else throw new NotImplementedException();
-
-        Console.WriteLine(string.Join(", ", clique.ToArray())
-            + $" ({clique.Count} vertices)");
-    }
-
-    private static void RunMCSFinder(string path, int index1, int index2, string algType, string comparison)
-    {
-        Graph[] graphs;
-        try
-        {
-            graphs = GraphLoader.Load(path);
-        }
-        catch (Exception e)
-        {
-            Console.WriteLine($"Couldn't load graphs from file {path}");
-            Console.WriteLine(e.Message);
-            return;
-        }
-
-        if (index1 > graphs.Length || index1 < 0
-            || index2 > graphs.Length || index2 < 0)
-        {
-            Console.WriteLine("Index out of bounds.");
-            return;
-        }
-
-        int[] subgraph1;
-        int[] subgraph2;
-        if (algType == "exact")
-        {
-            if (comparison == "vertices")
-            {
-                (subgraph1, subgraph2) = MCSFinder.FindFast(graphs[index1], graphs[index2], new MaxCliqueExactFinder());
-            }
-            else if (comparison == "vertices-then-edges")
-            {
-                (subgraph1, subgraph2) = MCSFinder.FindFast(graphs[index1], graphs[index2], new MaxCliqueExactFinder(), withEdges: true);
-            }
-            else throw new NotImplementedException();
-        }
-        else if (algType == "heuristic")
-        {
-            if (comparison == "vertices")
-            {
-                (subgraph1, subgraph2) = MCSFinder.FindFast(graphs[index1], graphs[index2], new MaxCliqueHeuFinder());
-            }
-            else if (comparison == "vertices-then-edges")
-            {
-                (subgraph1, subgraph2) = MCSFinder.FindFast(graphs[index1], graphs[index2], new MaxCliqueHeuFinder(), withEdges: true);
-            }
-            else throw new NotImplementedException();
-        }
-        else throw new NotImplementedException();
-
-        Console.WriteLine("MCS in graph 1: " + string.Join(", ", subgraph1));
-        Console.WriteLine("MCS in graph 2: " + string.Join(", ", subgraph2));
-        Console.WriteLine($"{subgraph1.Length} vertices");
+        return rootCommand;
```

Comment:
making a stupid error like giving a path to non-existing file causes very nasty error message about unhandled exception:
![image](https://github.com/KarolDenst/GraphAlgorithms/assets/54417806/5d937b09-a9c2-4ece-8f38-ce9a0063e8f4)
I think we should catch all exceptions and just print error message from the exception